ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java

```diff
@@ -150,14 +150,13 @@ public boolean bookingIdExistsOnPage(String id) {
         }
     }
 
-    public boolean isFirstBookingOK(ArrayList<Date> dates) {
+    public boolean isFirstBookingOK(ArrayList<LocalDate> dates) {
         try {
             WebElement firstVisitElement = findElement(FIRST_VISIT);
             if (firstVisitElement == null) {
                 return true;
             } else {
-                long dateparse = Date.parse((firstVisitElement.getAttribute("title")).toString());
-                Date date = new Date(dateparse);
+                LocalDate date = LocalDate.parse((firstVisitElement.getAttribute("title")).toString());
                 if (containsDate(dates, date)) {
```

Comment:
You can use List contains method, then you will not need containsDate or isSameDay methods